src/main/java/org/osc/controller/nsc/api/SampleSdnRedirectionApi.java

```diff
@@ -269,6 +263,18 @@ public InspectionPortElement getInspectionPort(InspectionPortElement inspectionP
         return ipEntity;
     }
 
+    public List<String> getInspectionPorts() throws Exception {
```

Comment:
same comment as above.

Also please group these  helper methods and move them to a section below all the interface methods and name the block something like 

// Support methods for API